collect_app/src/main/res/layout/automatic_project_creator_dialog_layout.xml

```diff
@@ -75,7 +75,8 @@
                 android:layout_height="wrap_content"
                 android:layout_marginTop="@dimen/margin_standard"
                 android:text="@string/configure_manually"
-                app:icon="@drawable/ic_edit"
+                android:textAppearance="?attr/textAppearanceBody1"
```

Comment:
Is this ok or should it be wrapped in something Collect-specific like the corresponding style?